File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/link/UpdateLinkParam.java
package com.cakk.domain.mysql.dto.param.link;

import java.util.List;

import lombok.Builder;

import com.cakk.domain.mysql.entity.shop.CakeShopLink;
import com.cakk.domain.mysql.entity.user.User;

@Builder
public record UpdateLinkParam(
	User user,
	Long cakeShopId,
	List<CakeShopLink> cakeShopLinks
) {
}

File: cakk-domain/mysql/src/main/java/com/cakk/domain/mysql/dto/param/link/UpdateLinkParam.kt
package com.cakk.domain.mysql.dto.param.link

import com.cakk.domain.mysql.entity.shop.CakeShopLink
import com.cakk.domain.mysql.entity.user.User

data class UpdateLinkParam(
	val user: User,
	val cakeShopId: Long,
	val cakeShopLinks: List<CakeShopLink>
)
